src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -62,7 +58,11 @@
                                   amount
                                   data-type
                                   fetch-criteria]]
-     {:fx [[:json-rpc/call
+     {:db (cond-> db
+            reset-collectibles?
+            (update-in [:wallet :ui :collectibles :reset-collectibles-request-ids]
+                       #(conj (or % #{}) request-id)))
```

Comment:
So, do we store the request-ids that were reset?

btw, consider using `fnil` on `conj` instead that `or`